lib/rules/function-calc-no-unspaced-operator/__tests__/index.mjs

```diff
@@ -263,32 +263,32 @@ testRule({
 			fixed: 'a { top: calc(2px + 1px); }',
 			description: 'no space before or after operator',
 			warnings: [
-				{ message: messages.expectedBefore('+'), line: 1, column: 18, endLine: 1, endColumn: 19 },
 				{ message: messages.expectedAfter('+'), line: 1, column: 18, endLine: 1, endColumn: 19 },
+				{ message: messages.expectedBefore('+'), line: 1, column: 18, endLine: 1, endColumn: 19 },
```

Comment:
[ask] Can we prevent this before/after order change?

Reply:
I now understand it seems quite difficult for now because an example of `2px+1px` is initially parsed as `2px` and `+1px` tokens. The algorithm is like this:

1. `2px+1px` → [`2px`, `+1px`] (the first and second operands)
2. [`2px`, `+1px`] → [`2px`, `+`, ` `, `1px`] with the `expectedAfter` message
3. [`2px`, `+`, ` `, `1px`] → [`2px`, ` `, `+`, ` `, `1px`] with the `expectedBefore` message

This is not a significant issue since the number of reported lint problems is unchanged. 👍🏼

Reply:
Correct :)